src/Mjolnir.ts

```diff
@@ -204,7 +208,8 @@ export class Mjolnir {
                 LogService.info("Mjolnir", `Command being run by ${event['sender']}: ${event['content']['body']}`);
 
                 await client.sendReadReceipt(roomId, event['event_id']);
-                return handleCommand(roomId, event, this);
+
+                return handleCommand(roomId, event, this, this.matrixCommandTable);
```

Comment:
Nit: Shouldn't `handleCommand` be a method of `matrixCommandTable`?

Reply:
Yep, but can't be until we have moved all the commands to the new table